akka-http-core/src/test/scala/akka/http/impl/engine/client/HostConnectionPoolSpec.scala

```diff
@@ -542,7 +542,7 @@ class HostConnectionPoolSpec extends AkkaSpecWithMaterializer(
         lazy val requestIn = TestPublisher.probe[RequestContext]()
         lazy val responseOut = TestSubscriber.probe[ResponseContext]()
 
-        protected val server: Flow[HttpRequest, HttpResponse, Future[Http.OutgoingConnection]]
+        protected def server: Flow[HttpRequest, HttpResponse, Future[Http.OutgoingConnection]]
```

Comment:
In Scala 2 lazy val cannot be in an abstract class, and in Scala 3 lazy val cannot override regular val. No problems with overriding def in either, it seems